frontend/lib/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.test.tsx

```diff
@@ -248,14 +248,13 @@ describe("StreamlitMarkdown", () => {
 
   it("doesn't render links when isButton is true", () => {
     // Valid markdown further restricted with buttons to eliminate links
-    const source = "Link: [text](www.example.com)"
-    const wrapper = render(
+    const source = "[Link text](www.example.com)"
+    render(
       <StreamlitMarkdown source={source} allowHTML={false} isLabel isButton />
     )
-    const container = wrapper.getByTestId("stMarkdownContainer")
-    const invalidTag = container.querySelector("a")
-    expect(invalidTag).toBeNull()
-    expect(container).toHaveTextContent("Link: ")
+    const tag = screen.getByText("Link text")
+    const isLinkTag = tag instanceof HTMLAnchorElement ? true : false
+    expect(isLinkTag).toBe(false)
```

Comment:
nit: I think this can just be `expect(tag instanceof HTMLAnchorElement).toBe(false)`, or alternatively

```typescript
    const isLinkTag = tag instanceof HTMLAnchorElement
    expect(isLinkTag).toBe(false)
```

if you want to keep the intermediate variable name for clarity